bin/gltf-pipeline.js

```diff
@@ -39,29 +41,35 @@ fs.readFile(gltfPath, function (err, data) {
     }
 
     var gltf = JSON.parse(data);
-    var stats = new OptimizationStatistics();
+    gltf = loadGltfUris(gltf, path.dirname(gltfPath), function(loadError) {
```

Comment:
I believe the common Node.js practice is to name this `err`, not `loadError`.  Any recent to deviate here?